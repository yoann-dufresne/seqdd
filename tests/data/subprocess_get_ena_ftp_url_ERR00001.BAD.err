  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   270  100   270    0     0   2711      0 --:--:-- --:--:-- --:--:--  2727
