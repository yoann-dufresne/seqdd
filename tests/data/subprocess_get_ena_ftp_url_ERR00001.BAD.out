<?xml version="1.0" encoding="UTF-8"?>
<ErrorDetails>
  <timestamp>1733215399814</timestamp>
  <status>400</status>
  <error>Bad Request</error>
  <message>Unknown accession format:ERR00001.BAD.</message>
  <path>/ena/browser/api/xml/ERR00001.BAD</path>
</ErrorDetails>
